generators/client/templates/vue/src/main/webapp/app/admin/user-management/user-management.vue.ejs

```diff
@@ -68,8 +68,7 @@
                                 <b-button v-on:click="prepareRemove(user)"
                                        variant="danger"
                                        class="btn btn-sm delete"
-                                       :disabled="username === user.login"
-                                       v-b-modal.removeUser>
```

Comment:
For some reason the same code works for entities, but not for the user management part. Therefore (in case no one else has a clue whats going on here) the modal is opened via script directly.